NativeAuthSampleApp/ProtectedAPIViewController.swift

```diff
@@ -26,24 +26,29 @@ import MSAL
 import UIKit
 
 class ProtectedAPIViewController: UIViewController {
-    let protectedAPIUrl = "Enter_the_Protected_API_Full_URL_Here"
-    let protectedAPIScopes = ["Enter_the_Protected_API_Scopes_Here"]
+
+    let protectedAPIUrl1: String? = "Enter_the_Protected_API_Full_URL_Here"
+    let protectedAPIScopes1: [String] = ["Enter_the_Protected_API_Scopes_Here"]
+    
+    // Enter the second protected API info Here, if you have one.
+    let protectedAPIUrl2: String? = nil
```

Comment:
I left the second API configuration empty because some users (maybe most?) won't use it, so it becomes clear that an empty config is acceptable if you only have one API to test with.

Reply:
Updated. Now we're aligned to Android and the check that API1 or API2 details are there, works correctly